packages/docusaurus/src/client/PendingNavigation.js

```diff
@@ -94,7 +96,7 @@ class PendingNavigation extends React.Component {
     this.clearProgressBarTimeout();
     this.progressBarTimeout = setTimeout(() => {
       clientLifecyclesDispatcher.onRouteUpdateDelayed({
-        location: this.props.location,
+        location: this.state.location,
```

Comment:
There's no `this.state.location`

Reply:
:sweat_smile: I missed a reference to the old state value... Now it's fixed.

Reply:
Thank you! 😄